src/selectors/node-types.js

```diff
@@ -40,3 +41,20 @@ export const getNodeTypes = createSelector(
       nodeCount: typeNodeCount[id]
     }))
 );
+
+/**
+ * Get formatted list of sections
+ */
+export const getNodeSections = createSelector(
+  [getNodeTypeSections, getNodeTypes],
+  (sections, types) => {
+    const result = [];
+    for (const name of Object.keys(sections)) {
+      result.push({
+        name,
+        types: sections[name].map(id => types.find(type => type.id === id))
```

Comment:
Will this not include some undefined items?

Reply:
Now removed so I'll resolve this one.